legacy/application/models/airtime/map/CcPlaylistTableMap.php

```diff
@@ -56,6 +56,8 @@ public function buildRelations()
     {
         $this->addRelation('CcSubjs', 'CcSubjs', RelationMap::MANY_TO_ONE, array('creator_id' => 'id', ), 'CASCADE', null);
         $this->addRelation('CcShow', 'CcShow', RelationMap::ONE_TO_MANY, array('id' => 'autoplaylist_id', ), 'SET NULL', null, 'CcShows');
+        $this->addRelation('CcShow', 'CcShow', RelationMap::ONE_TO_MANY, array('id' => 'intro_playlist_id', ), 'SET NULL', null, 'CcShows');
+        $this->addRelation('CcShow', 'CcShow', RelationMap::ONE_TO_MANY, array('id' => 'outro_playlist_id', ), 'SET NULL', null, 'CcShows');
```

Comment:
Please update the legacy schema.xml file and regenerate these files with the make propel-gen command. You will have to run a bunch of make -C legacy format and pre-commit run --all to fix the formatting.

Reply:
Thanks for the hint, i was not aware these were generated instead of modelled by hand.